### ISSUE TITLE
I think there was a minor error in the image reading

### ISSUE THREAD

**Issue by ghost** [TreB1eN/InsightFace_Pytorch#92]

**Comment by ghost** [comment]
Hi, thank you for providing the version of PyTorch. But I find that at the beginning of the training, the accuracy of the three validation datasets was very low. (I use `MobileFaceNet`, `ArcFace` loss and train on `CASIA-WebFace`, and the validation accuracy on `lfw` at the beginning epoch of training only `70%`).

Then I find the code `img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)` in `data_pipe.py` transforms the `RGB` to `BGR`. But in `PIL`, the image has the mode `RGB` and `transforms.ToTensor()` can process `PIL image` directly. So I think this transform is useless. 

And I add one line of code(`carray = carray[:, ::-1, :, :].copy()`) in function `evaluate` to change the `BRG` to `RGB`. The accuracy of validation at the beginning of the train is higher. 
<img width="798" alt="image" src="https://user-images.githubusercontent.com/35477594/62295878-9b9c7580-b4a0-11e9-8303-58ef467b8c91.png">

So I think maybe should not do the transform `RGB` to `BGR`.
